src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -131,7 +131,7 @@ public string BuildTrackFileName(List<Track> tracks, Artist artist, Album album,
 
                 var component = ReplaceTokens(splitPattern, tokenHandlers, namingConfig).Trim();
 
-                component = FileNameCleanupRegex.Replace(component, match => match.Captures[0].Value[0].ToString());
```

Comment:
Rather than this change, would it be better to modify the 'CleanFolderName' method to trim leading "." characters? Then if a user would like to keep the leading "." they can use _{Album Title}_, but trimming is available with _{Album CleanTitle}_ or any of the other 'clean' tags.